tools/PingEnumerator.py

```diff
@@ -34,11 +34,12 @@
 
         if newPing.initialize() == True:
             try:
-                if newPing.get_device_id() != None and newPing.get_fw_version() != None:
-
+                if newPing.get_device_id() != None and newPing.get_firmware_version() != None:
                     # ex Ping1D-id-43-t-1-m-1-v-3.19
-                    description = "/dev/serial/ping/Ping1D-id-%s-t-%s-m-%s-v-%s.%s" % (newPing.device_id, newPing.device_type, newPing.device_model, newPing.fw_version_major, newPing.fw_version_minor)
-                    print("Found Ping1D (ID: %d) at %s" % (newPing.device_id, dev))
+                    description = "/dev/serial/ping/Ping1D-id-%s-t-%s-m-%s-v-%s.%s" % (
+                    newPing._device_id, newPing._device_type, newPing._device_model, newPing._firmware_version_major,
```

Comment:
let's not access the private vars, but store them here with the return of getters.